config/locales/en.yml

```diff
@@ -1824,6 +1824,7 @@ en:
           information: "Pending should only be used for incomplete (eg, unpaid) registrations - please use the Waiting List if your competition is too full to accept new competitors."
         waitlist:
           title: "Waiting List Registrations"
+          skipped_warning: "You've skipped %{count} registration(s) on the Waiting List. Are you sure you want to approve the other registration(s) instead?"
```

Comment:
Why did you decide against the pluralization feature (separate sub-keys for `one`, `many`, `other`, etc) in this case?

Reply:
The second "registration(s)" is independent of the first. It's also not entirely clear how it should work. If you take positions 2, 3, and 8 on the waitlist, does "other registration(s)" refer to just 8, or all 3? You're accepting 8 instead of 1, vs accepting 2, 3, and 8 instead of 1, 2, and 3.
I also saw some other "(s)"s in the file so assumed it was ok.

I can pluralize the first one if you want. And the second if I know how it should work.

Reply:
I guess another solution would be to re-phrase the second sentence.

Reply:
Mh, no hard feelings here. Your argument is strong enough to convince me that we don't need to go the sophisticated route :)